pkg/backporter/backporter.go

```diff
@@ -370,3 +413,80 @@ func GetClonesHandler(client bugzilla.Client) HandlerFuncWithErrorReturn {
 		return nil
 	}
 }
+
+// CreateCloneHandler will create a clone of the specified ID and return success/error
+func CreateCloneHandler(client bugzilla.Client) HandlerFuncWithErrorReturn {
+	return func(w http.ResponseWriter, req *http.Request) error {
+		req.ParseForm()
+		bugID, err := strconv.Atoi(req.FormValue("ID"))
+		if err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+			wpErr := writePage(w, "Error", errorTemplate, fmt.Sprintf("Invalid bug id! - Bug#%d : %v", bugID, err))
```

Comment:
Please don't use exclamation marks in messages, there is no reason to have the software yell at the user